auth0/src/main/java/com/auth0/android/result/Credentials.java

```diff
@@ -103,6 +103,13 @@ public String getRefreshToken() {
         return refreshToken;
     }
 
+    /**
+     * Getter for the amount of seconds since the tokens were issued in which they will be deemed invalid.
+     * To know precisely when this is going to happen you need to save the time every time you request a new pair of tokens.
+     *
+     * @return the seconds since the tokens where issued in which they will be deemed invalid.
```

Comment:
minor comment, but this is like a TTL, maybe the method and return description is easier to understand if you just say that. 
or have a native english speaker help here, this doesn't sound that well